### ISSUE TITLE
Duplicated or similar terms

### ISSUE THREAD

**Issue by gkellogg** [w3c/rdf-star-wg#37]
Terms created via `<dfn` become exported and can be automatically reconciled by other specifications. Between RDF Concepts and Semantics a number of terms are created, and sometimes overlap.

* It's possible to "define" a term but reference a term from another spec using `data-cite`. For example, `<dfn data-cite="RDF12-CONCEPTS">RDF graph</dfn>`, which styles the term in the defining spec, and allows simple references such as `<a>RDF graph</a>`.
* Terms intended to be used locally can be declared to not be exported, for example `<dfn class="no-export lint-ignore" data-dfn-for="rdf">invalid</dfn>` (which comes from https://github.com/w3c/rdf-semantics/issues/13).
* Former terms can be demoted, but should have an anchor so that any existing references will continue to resolve. For example `<dfn class="export">invalid</dfn><span id="dfn-invalid.x"><!-- obsolete term --></span>`.

Given the potential range of updates, we should consider what to do about the following terms:

* `Denote` – defined in [RDF12-CONCEPTS](https://w3c.github.io/rdf-concepts/spec/#dfn-denote) and in [RDF12-Semantics](https://w3c.github.io/rdf-semantics/spec/#dfn-denote). Also [Denotation](https://w3c.github.io/rdf-semantics/spec/#dfn-denotation) in Semantics.
* `Entailment` – defined in [RDF12-CONCEPTS](https://w3c.github.io/rdf-concepts/spec/#dfn-entailment) and as [Entail](https://w3c.github.io/rdf-semantics/spec/#dfn-entail) in Semantics
* `Equivalent` – Defined as [Equivalence](https://w3c.github.io/rdf-concepts/spec/#dfn-equivalence) in Concepts and [Equivalent](https://w3c.github.io/rdf-semantics/spec/#dfn-equivalent) in Semantics.
* `Referent` – defined in [RDF12-CONCEPTS](https://w3c.github.io/rdf-concepts/spec/#dfn-referent) and in [RDF12-SEMANTICS](https://w3c.github.io/rdf-semantics/spec/#dfn-referent)

Many other terms in Semantics are exported but not used, at least in RDF Semantics (e.g., [`rdfs13`](https://w3c.github.io/rdf-semantics/spec/#dfn-rdfs13). We might consider just making these "no-export" and just preserve the anchor.

Thus far, SPARQL doesn't export any terms, but probably will at some point.

* [RDF Term](https://w3c.github.io/sparql-query/spec/#defn_RDFTerm)
* [Simple Literal](https://w3c.github.io/sparql-query/spec/#defn_SimpleLiteral) – Obsolete now, but matches [simple literal](https://w3c.github.io/rdf-concepts/spec/#dfn-simple-literal) in RDF Concepts.
* [RDF Dataset](https://w3c.github.io/sparql-query/spec/#defn_RDFDataset)

**Comment by afs** [comment]
`RDF Term` and `RDF dataset` are now defined in RDF 1.1 Concepts. (`RDF Dataset` is a different definition).

Such terminology wasn't available for SPARQL 1.0 or SPARQL 1.1.

**Comment by hartig** [comment]
Indeed, the definition of 'RDF dataset' in RDF 1.1 and in SPARQL 1.1 is slightly different. The difference is in terms of what can be a name of the named graphs that are contained in such a dataset. For the notion of [RDF dataset in RDF 1.1](https://www.w3.org/TR/rdf11-concepts/#dfn-rdf-dataset), the graph names can be IRIs or blank nodes, whereas for the notion of [RDF dataset in SPARQL 1.1](https://www.w3.org/TR/sparql11-query/#sparqlDataset), the graph names can only be IRIs. I guess that this difference is also something we want to get rid of.

**Comment by pchampin** [comment]
> Recorded as : [w3c/sparql-query#28](https://github.com/w3c/sparql-query/issues/28)

and the other part (about the overlap between RDF-CONCEPTS and RDF-SEMANTICS) has been raised in https://github.com/w3c/rdf-concepts/issues/118 .

So I believe that this issue can be closed.

**Comment by pfps** [comment]
I agree.

**Comment by gkellogg** [comment]
Seems to have been resolved.
